### DISCUSSION TITLE
include some theme/style examples in docs

### DISCUSSION THREAD

**Discussion by dyllandry** [retypeapp/retype#5] [Q&A] [answered]
Is there a way I can see what's possible with the site layout/theme/styles without downloading and trying the project out for myself? Maybe some examples or something. Thank you!

**Comment by geoffreymcgill** [comment, marked as answer]
Hi. At this point, Retype does not include optional layouts or themes or much in the way of customizable styles. 

The main priority for the moment is building functionality. We do anticipate adding some `layout` options and more flexibility soon, but Retype is still going to be fairly _opinionated_ about what will be possible. The goal is much more focused on generating a highly functional, usable, and modern site... than on providing theme or style flexibility.

Supporting dark mode also complicates styling, at least if you want to do it well.

Retype helps content developer focus on the quality of their content rather than dealing with the complexities of building a site.

There is some flexibility with the `identity` [options](https://retype.com/project_configuration/#options), although limited to `title`, using an image for the `logo`, customizing the `label`, and few other options. Not really what I would call customizable styles, but it's there.

If you have specific layout/theme/style requirements or suggestions, just let us know and we will certainly investigate.

Thanks for the interest in Retype.

**Update:** Converted to a discussion from issue #4.
